resource/create.go

```diff
@@ -0,0 +1,19 @@
+package resource
+
+import (
+	"github.com/sul-dlss-labs/taco/generated/restapi/operations"
+)
+
+func LoadParams(id string, params operations.DepositResourceParams) interface{} {
```

Comment:
I believe this is only used once in the CreateResource handler. Can we consolidate to avoid indirection?